src/getstatusinternaloptions.ts

```diff
@@ -1,16 +1,16 @@
-// tslint:disable:member-access
-
-import { OrchestrationRuntimeStatus } from "./classes";
-
-/** @hidden */
-export class GetStatusOptions {
-    instanceId?: string;
-    taskHubName?: string;
-    connectionName?: string;
-    showHistory?: boolean;
-    showHistoryOutput?: boolean;
-    createdTimeFrom?: Date;
-    createdTimeTo?: Date;
-    runtimeStatus?: OrchestrationRuntimeStatus[];
-    showInput?: boolean;
-}
+// tslint:disable:member-access
+
+import { OrchestrationRuntimeStatus } from "./classes";
+
+/** @hidden */
+export class GetStatusInternalOptions {
```

Comment:
This one should def be an interface lol (especially since it's internal). And I'd move it directly into `durableorchestrationclient.ts`

Reply:
Done